tests/DatapackageTest.php

```diff
@@ -476,6 +476,29 @@ public function testCreateEditDatapackageDescriptor()
         $zip->close();
         unlink($filename);
         $tempdir = $tempdir.DIRECTORY_SEPARATOR;
+
+        //after saving to disk, the paths are updated
+        $expectedDatapackageDescriptor = (object) [
+            'name' => 'my-datapackage-name',
+            'resources' => [
+                (object) [
+                    'name' => 'my-default-resource',
+                    'path' => ["resource-0-data-0", "resource-0-data-1"],
+                ],
+                (object) [
+                    'name' => 'my-renamed-tabular-resource',
+                    'path' => "resource-1.csv",
```

Comment:
This is where were should test for paths that are also file name and URI maybe

Reply:
Agreed. Thanks for taking the time to review this!

Reply:
Thanks to you! This is all great and the needed changes are not many. Good work